lib/drivers/steem.js

```diff
@@ -0,0 +1,81 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * Steemdb driver. Supports total and circulating supply
+ * for tokens STEEM on STEEM blockchain.
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class Steem extends Driver {
+  constructor(options) {
+    super(
+      {
+        timeout: 100, // 10 requests per second
+        supports: {
+          native: true,
+          circulating: true,
+          blockchains: ['Steem'],
+        },
+      },
+      options,
+    );
+  }
+
+  /** get total supply for native token
+   *
+   * @augments Driver.fetchTotalSupply
+   * @async
+   */
+  async fetchTotalSupply() {
+    const data = await this.request({
+      url: 'https://steemdb.io/api/props',
+      rejectUnauthorized: false,
+    });
+    const record = data.find((item) => item._id && item._id.$oid === '5f088eda7c9c237b7d490dac');
```

Comment:
Can you change this one as well like the hive driver? That it will get the latest prop based on the `time` property? Otherwise this will not work anymore over time